doc/api/pyplot_api.rst

```diff
@@ -0,0 +1,264 @@
+******
+pyplot
+******
+
+.. currentmodule:: matplotlib.pyplot
+
+.. automodule:: matplotlib.pyplot
+    :no-members:
+
+Information
+===========
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    colormaps
+    get_plot_commands
+
+Global functions
+================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    findobj
+    install_repl_displayhook
+    ioff
+    ion
+    isinteractive
+    pause
+    rc
+    rc_context
+    rcdefaults
+    show
+    switch_backend
+    uninstall_repl_displayhook
+
+Any artist
+==========
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    getp
+    setp
+    xkcd
+
+Making figures
+=============
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    figure
+    subplots
+
+Working with figures
+====================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    clf
+    close
+    connect
+    disconnect
+    draw
+    fignum_exists
+    figure
+    gcf
+    get_current_fig_manager
+    get_figlabels
+    get_fignums
+    ginput
+    savefig
+    waitforbuttonpress
+
+Putting things in figures
+=========================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    figimage
+    figlegend
+    figtext
+    suptitle
+
+Working with axes
+=================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    axes
+    box
+    delaxes
+    gca
+    sca
+    subplot
+    subplots
+    subplot2grid
+    subplot_tool
+    subplots_adjust
+    tight_layout
+    twinx
+    twiny
+
+Working with axis
+=================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    rgrids
+    thetagrids
+    xlim
+    ylim
+    xticks
+    yticks
+
+Current image
+=============
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    gci
+
+Plotting functions
+==================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    acorr
+    angle_spectrum
+    annotate
+    arrow
+    autoscale
+    axhline
+    axhspan
+    axis
```

Comment:
Those are no plotting functions: axis, cla, grid, sci, polar, xlabel, ylabel, set_cmap, title, xscale, yscale etc.

Reply:
Ok, thats true. They are in the plotting section of pyplot though and I haven't look through that part in much detail.